client/styles/index.scss

```diff
@@ -2,7 +2,7 @@
 @import "./variables.scss";
 
 // Import all style of Bulma framework.
-@import "../../../node_modules/bulma/bulma.sass";
+@import "../../node_modules/bulma/bulma.sass";
```

Comment:
Can't you use ~?

Reply:
I check that and no. Webpack just can't find this import with '~'.